runtime/jcl/common/java_lang_StackWalker.cpp

```diff
@@ -146,6 +147,8 @@ Java_java_lang_StackWalker_walkWrapperImpl(JNIEnv *env, jclass clazz, jint flags
 	if (NULL == walkerMethodChars) { /* native out of memory exception pending */
 		return NULL;
 	}
+	/* Ensure userData1/2 used by stackFrameFilter function is set properly. */
+	walkState->userData1 = NULL;
```

Comment:
Please explain why this is appropriate. Should line 156 not be here instead?
```c
	walkState->userData1 = (void *)(UDATA)flags;
```
Same question for continuations (lines 216-220).

Reply:
You are right, I placed the NULL to avoid conflict with getCallerClass call, but given #18868 is already merged, it should be fine to move the assignment earlier.

I will update the PR and run local testing before asking for another round of review.

Reply:
local testing passed, @keithc-ca can you please take another look